dev/tasks/utils/cs-build-entry.js

```diff
@@ -0,0 +1,16 @@
+// jshint ignore: start
+
+import FileUploader from '@ckeditor/ckeditor-cloudservices-core/src/uploadgateway/fileuploader';
+import UploadGateway from '@ckeditor/ckeditor-cloudservices-core/src/uploadgateway/uploadgateway';
+
+window.CKEDITOR = ( window.CKEDITOR || {} );
+
+window.CKEDITOR.cloudservices = {
```

Comment:
It should be a camelCased property name. I like the idea about putting it statically to `CKEDITOR` namespace. Later, if more plugins uses CS, we could create a `cloudservices` plugin that sets up this namespace.